wormdatamodel/data/recording.py

```diff
@@ -128,10 +130,10 @@ class recording:
     
     # Optogenetics
     optogeneticsN = 0
-    optogeneticsFrameCount = np.zeros(optogeneticsN, dtype=np.int)
-    optogeneticsNPulses = np.zeros(optogeneticsN, dtype=np.int)
-    optogeneticsRepRateDivider = np.zeros(optogeneticsN, dtype=np.int)
-    optogeneticsNTrains = np.zeros(optogeneticsN, dtype=np.int)
+    optogeneticsFrameCount = np.zeros(optogeneticsN, dtype=np.int64)
+    optogeneticsNPulses = np.zeros(optogeneticsN, dtype=np.int64)
+    optogeneticsRepRateDivider = np.zeros(optogeneticsN, dtype=np.int64)
+    optogeneticsNTrains = np.zeros(optogeneticsN, dtype=np.int64)
```

Comment:
Numpy deprecated this behavior a while back (first a soft warning for several versions; then finally a hard removal that caused some of my errors)

There's actually 2 parts to it as well

i) `np.int` has become `np.integer` and is the 'base' data type of all specific `np.int<number of bits>` types
ii) they no longer allow `dtype=np.integer` which always defaulted a system specific `np._int` value - the modern recommendation is to set it to an explicit size